integration-tests/src/test/java/oracle/kubernetes/operator/ITOperator.java

```diff
@@ -144,202 +134,192 @@ public static void staticUnPrepare() throws Exception {
     boolean testCompletedSuccessfully = false;
     try {
-      // create domain3
-      domain3 = testDomainCreation(domain3YamlFile);
-      domain3.verifyDomainCreated();
-      testBasicUseCases(domain3);
-      testWLDFScaling(operator1, domain3);
+      // create domain
+      domain = TestUtils.createDomain(domainOnPVUsingWDTYamlFile);
+      domain.verifyDomainCreated();
+      testBasicUseCases(domain);
+      testWLDFScaling(operator2, domain);
+      // TODO: Test Apache LB
+      // domain.verifyAdminConsoleViaLB();
       testCompletedSuccessfully = true;
     } finally {
-      if (domain3 != null && (JENKINS || testCompletedSuccessfully)) {
-        domain3.destroy();
+      if (domain != null && (JENKINS || testCompletedSuccessfully)) {
+        logger.info("About to delete domain: " + domain.getDomainUid());
+        TestUtils.deleteWeblogicDomainResources(domain.getDomainUid());
+        TestUtils.verifyAfterDeletion(domain);
       }
     }
 
-    logger.info("SUCCESS - test3CreateDomainInTest1NS");
-  }
-  /**
-   * Create operator managing test2 namespace
-   *
-   * @throws Exception
-   */
-  @Test
-  public void test4CreateAnotherOperatorManagingTest2NS() throws Exception {
-    Assume.assumeFalse(QUICKTEST);
-
-    logTestBegin("test4CreateAnotherOperatorManagingTest2NS");
-    logger.info("Creating Operator & waiting for the script to complete execution");
-    // create operator2
-    operator2 = TestUtils.createOperator(op2YamlFile);
-    logger.info("SUCCESS - test4CreateAnotherOperatorManagingTest2NS");
+    logger.info("SUCCESS - " + testMethodName);
   }
 
   /**
    * Create two operators if they are not running. Create domain domain4 with dynamic cluster in
-   * default namespace, managed by operator1 Create domain domain5 with Configured cluster using WDT
-   * in test2 namespace, managed by operator2 Verify scaling for domain5 cluster from 2 to 3 servers
-   * and back to 2, plus verify no impact on domain4 Cycle domain4 down and back up, plus verify no
-   * impact on domain5 shutdown by deleting both domain4 and domain5 CRD's
+   * default namespace, managed by operator1. Create domain domain5 with Configured cluster using
+   * WDT in test2 namespace, managed by operator2 Verify scaling for domain5 cluster from 2 to 3
+   * servers and back to 2, plus verify no impact on domain4 Cycle domain4 down and back up, plus
+   * verify no impact on domain5 shutdown by deleting both domain4 and domain5 CRD's
+   *
+   * <p>ToDo: configured cluster support is removed from samples, modify the test to create
    *
    * @throws Exception
    */
   @Test
-  public void test5CreateConfiguredDomainInTest2NS() throws Exception {
+  public void testTwoDomainsManagedByTwoOperators() throws Exception {
     Assume.assumeFalse(QUICKTEST);
-
-    logTestBegin("test5CreateConfiguredDomainInTest2NS");
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
```

Comment:
To help avoid cut-and-paste code for logging the begin/end of each test, it looks like you can use an alternate approach to get the current method:
```
  private static final int CALLER_INDEX;
  static {
    int i = 0;
    for (StackTraceElement ste : Thread.currentThread().getStackTrace()) {
      i++;
      if (ste.getClassName().equals(ITOperator.class.getName())) {
        break;
      }
    }
    CALLER_INDEX = i;
  }

  private static String currentTest = "";

  public static void logTestBegin() {
    currentTest = Thread.currentThread().getStackTrace()[CALLER_INDEX].getMethodName();
    System.out.println("Test '" + currentTest + "' - BEGIN");
  }

  public static void logTestSuccess() {
    System.out.println("Test '" + currentTest + "' - SUCCESS");
  }

  public static void logTestInfo(String s) {
    System.out.println("Test '" + currentTest + "' - INFO: " + s);
  }

  /*
  // Sample usage
  public void someTest() {
    logTestBegin();
    logTestInfo("Something something.");
    logTestSuccess();
  }
  */
```
This won't always work on all JVMs according to 'https://stackoverflow.com/questions/442747/getting-the-name-of-the-currently-executing-method/8592871#8592871', but since the operator project always uses the Sun JVM it should be fine.

Reply:
I tried the above code, its not working Tom, its printing invoke0 for the method name.